depthimage_to_navscan_rgbd/src/depthimage_to_navscan_rgbd.cpp

```diff
@@ -24,6 +24,13 @@ int main(int argc, char** argv)
         ROS_FATAL("[Depthimage to Navscan RGBD] could not read frame_id from parameter server");
         return 1;
     }
+    
+    // Optional
+    double rate = 10;
```

Comment:
Default behaviour should be to go as fast as possible. i.e. the rate at which images come in. 
I would add a check to see if rate was set before the sleep in the loop.

Reply:
When setting the rate high enough we achieve this